lib/fauna/query.rb

```diff
@@ -473,8 +499,58 @@ def self.with_auto_vars(n_vars)
     #
     # This ensures that a single value passed is not put in an array, so
     # <code>query.add([1, 2])</code> will work as well as <code>query.add(1, 2)</code>.
-    def self.varargs(values)
+    def varargs(values)
       values.length == 1 ? values[0] : values
     end
   end
+
+  ##
+  # Build a query expression.
```

Comment:
Expand documentation with the requirements for using this and example code blocks. The end user of this should understand how to use it without assistance or having to reverse the code.